packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -60,11 +60,14 @@ gmaps.MapOptions _configurationAndStyleToGmapsOptions(
     options.zoomControl = configuration.zoomControlsEnabled;
   }
 
-  if (configuration.scrollGesturesEnabled == false ||
-      configuration.zoomGesturesEnabled == false) {
-    options.gestureHandling = 'none';
+  if (configuration.webGestureHandling != null) {
+    options.gestureHandling = configuration.webGestureHandling!.name;
```

Comment:
Something like this:

```suggestion
  if (configuration.scrollGesturesEnabled == false ||
      configuration.zoomGesturesEnabled == false) {
    html.window.console.warn('`scrollGesturesEnabled` and `zoomGesturesEnabled` are now ignored on the web. Use `webGestureHandling: WebGestureHandling.none` instead.');
  }

  if (configuration.webGestureHandling != null) {
    options.gestureHandling = configuration.webGestureHandling!.name;
```